### PR TITLE
feat - added homepage basic structure

### PR DESCRIPTION
**Pull request by akash-1318**
## Added basic structure of homepage - 
I have build the basic structure for homepage it does not have any functionality yet.

## Files to check - 
1. src/pages/homepage/home.js
2. src/pages/homepage/home.css
3. src/components/post/post.js
4. src/components/post/post.css
5. src/components/new-post/new-post.js
6. src/components/new-post/new-post.css
7. src/components/follow-suggestion/follow-suggestion.js
8. src/components/follow-suggestion/follow-suggestion.css
9. src/components/aside/aside.js
10. src/components/aside/aside.css
Screenshot 2022-06-17 at 5 54 54 PM

### CODE REVIEW

File: src/styles/common.css [hunk 1]
```diff
@@ -24,7 +28,9 @@
     --slider-thumb-bg-color:#636363;
     --white-color : #ffffff;
     --tomato-color : #ff6347;
-    --secondry-light-color : #ffb4dd;
+    --secondry-light-color : #aeb9ff;
+    /* --secondry-light-color : #ffd085; */
+    /* --secondry-light-color : #ffb4dd; */
```
Comment (srv-pranjal) [inline src/styles/common.css:33]: remove these comments if not required for any particular reason

**Review by srv-pranjal — APPROVED** [review]
UI looks great.
Good folder structure.
Left one suggestion rest everything looks good

File: src/components/follow-suggestion/follow-suggestion.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    cursor: pointer;
+}
+
+.see__more {
+    font-size: 1.8rem;
+    margin: 1.5rem 0rem;
+    text-decoration: underline;
+    text-underline-offset: 2px;
+    color: var(--primary-color);
+    cursor: pointer;
```
Comment (irajbhinde) [inline src/components/follow-suggestion/follow-suggestion.css:102]: small thing but cursor pointer is in entire container, you can either center the see more or hit pointer only on text

File: src/components/aside/aside.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import "./aside.css"
+import logo from "../../assets/images/logo.png"
+
+function Aside () {
+    return(
+        <aside className="aside">
+            <div className="aside__header">
+                <img src={logo}/>
+                <p>BeLoud</p>
```
Comment (irajbhinde) [inline src/components/aside/aside.js:9]: You can add a cursor pointer & a link here if required

**Review by irajbhinde — COMMENTED** [review]
Code looks good & UI looks clean, especially the landing page. Kudos!

File: src/components/aside/aside.js (hunk 3 above)
Comment (akash-1318) [reply src/components/aside/aside.js:9]: Yeah will do it.

**Pull request merged**